bugsnag-plugin-android-okhttp/src/test/java/com/bugsnag/android/okhttp/NetworkBreadcrumbIntegrationTest.kt

```diff
@@ -1,63 +0,0 @@
-package com.bugsnag.android.okhttp
```

Comment:
This has been split into various integration tests which will be added in a separate changeset for ease of review